src/js/components/FileInput/__tests__/__snapshots__/FileInput-test.js.snap

```diff
@@ -1239,7 +1239,7 @@ exports[`FileInput messages 1`] = `
       <span
         class="c4 "
       >
-        Drag and drop
+        Drop file here or
```

Comment:
Any idea why this changed?

Reply:
Yes. This was a case where the way FileInput was written it takes the messages object passed in and only uses that as a whole. It no longer has whatever was in defaultProps.messages. So if you pass in a sparse messages object (ie. missing some messages) it was using the hardcoded default rather than the message in the default props. This was the only case where the hardcoded was different than default props.

The way this works, the default always comes from default.json (although the format function supports a defaultMessage member similar to intl.formatMessage()). And personally I thought that hidden default was not very useful :)